module-utils/utility/Utils.hpp

```diff
@@ -158,6 +158,11 @@ namespace utils
         if (value.empty()) {
             return {};
         }
+
+        if constexpr (std::is_same<uint8_t, T>::value || std::is_same<int8_t, T>::value) {
+            return std::stoi(value);
```

Comment:
You should probably check for exception in case of stoi.

Reply:
Added, thanks!